docker-compose.yml

```diff
@@ -21,6 +21,46 @@ services:
             - ./api-dummy/public:/srv/app/public
         depends_on:
             - api-dummy-fpm
+    dashboard-events:
+        build:
+            context: ./dashboard
+        volumes:
+            - ./dashboard:/srv/app
+            - /srv/app/public/bundles
+            - /srv/app/var
+            - /srv/app/vendor
+        environment:
+            - APP_ENV=dev
+            - APP_SECRET=f6a87c2e031ecadc0d7eb442729435c7
+            - RABBITMQ_URL=amqp://guest:guest@event-bus:5672
+        command: bash -c '/utils/wait-for-port event-bus 5672 && bin/console rabbitmq:consumer events'
```

Comment:
seems easy enough to consume the events. Where to put them to make them available to the UI is a design decision